### DISCUSSION TITLE
Can I update season posters with plex-meta-manager?

### DISCUSSION THREAD

**Discussion by cpt-kuesel** [meisnate12/Plex-Meta-Manager#110] [Q&A] [answered]
Hi all,

I want to update posters for a few shows and ideally use plex-meta-manager for that.

I created a collection consisting of one show and set it to hide the collection, so that the show will still show up normally.
My assets directory looks like this:
config
-> assets
-> -> Bleach
-> -> -> Bleach (2004)
-> -> -> -> Season 01
-> -> -> -> -> poster.png // season 01 poster
-> -> -> -> Season 02
-> -> -> -> -> poster.png // season 02 poster
-> -> -> -> Season 03
-> -> -> -> -> poster.png // season 03 poster
[...] // repeat for all other seasons
-> -> -> -> poster.png // show poster
-> -> -> poster.png // collection poster

So far plex-meta-manager only updates the poster of the collection and of the show itself. 

Is it possible to update the season posters and I overlooked a setting?
If it is not possible atm, is it a reasonable feature request?

**Comment by meisnate12** [comment, marked as answer]
Season posters were added in v1.4.0 you can see how to use them [here](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Settings-Attributes#image-asset-directory)

**Reply by cpt-kuesel** [reply]
perfect, thank you!
